docs/guides/10-add-legends.md

```diff
@@ -10,36 +10,95 @@ If you completed the XXX guide, the map below will look familiar. In that guide,
 
 <div class="example-map">
     <iframe
-        id="accidents-all"
-        src="/developers/carto-vl/examples/maps/guides/legends/accidents-all-legend.html"
+        id="guides-legend-step-3"
+        src="/developers/carto-vl/examples/maps/guides/add-legends/step-3.html"
         width="100%"
         height="500"
-        style="margin: 20px auto !important"
         frameBorder="0">
     </iframe>
 </div>
 
 At the end of this guide, we also provide a series of examples, that are meant to serve as legend "building blocks" that you can take and begin to customize on top of for a variety of map types.
 
-### Getting Started
+### Getting started
 
 The map below is the same one as above, a category map that symbolizes US rail accidents by reported weather conditions. Unlike the map above, you will notice that this map has a legend box with a title ("Rail Accidents by Weather") in the right hand corner, but is missing the legend information to help interpret what weather type each color on the map represents. 
 
-Let's add that information!
-
-To get started, copy and paste the code for this map and save it as `accidents.html`:
-
 <div class="example-map">
     <iframe
-        id="population-density-buckets-numeric"
-        src="/developers/carto-vl/examples/maps/guides/legends/accidents-all-no-legend.html"
+        id="guides-legend-step-1"
+        src="/developers/carto-vl/examples/maps/guides/add-legends/step-1.html"
         width="100%"
         height="500"
         style="margin: 20px auto !important"
         frameBorder="0">
     </iframe>
 </div>
-<a href="/developers/carto-vl/examples#example-accidents-all---no-legend">Copy my source code!</a>
+
+Let's add that information!
+
+To get started, copy and paste the code for this map and save it as `accidents.html`:
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Rail accident weather | CARTO</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8">
+    <script src="../../../dist/carto-vl.js"></script>
+    <script src="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.js"></script>
```

Comment:
Patched MGL :eyes:

Reply:
👍 good catch.  I have replaced all v0.48 references with 0.50 and also fixed some bad hardcoded urls for these guides: '10-add-legends' and '04-data-driven-visualizations-part-1'